### PR TITLE
API examples: Improved wording in examples/README.md

### PR DESCRIPTION
**Pull request by s-l-teichmann**
Repair link to PURL searcher.

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # API examples
 
-A currenty very short list how to use `github.com/csaf-poc/csaf_distribution` as a library:
+A currenty very short list how to use `github.com/csaf-poc/csaf_distribution` as a library.
+As the API is currently WIP these are are likely to be modified or extented.
 
-* (purl_searcher)[./purl_searcher/main.go] is a tool to search through local advisories. It finds PURLs based on the product ID of an advisory.
+* [purl_searcher](./purl_searcher/main.go) is a tool to search PURLs in local advisories. It finds them by the product IDs.
```
Comment (ThomasJunk) [inline examples/README.md:6]: ..is a tool to search for(?) PURLs ..

**Review by ThomasJunk — APPROVED** [review]
LGTM

**Review by JanHoefelmeyer — APPROVED** [review]
LGTM

**Review by ThomasJunk — APPROVED** [review]
LGTM

**Pull request merged**